app/src/main/java/de/berlindroid/zepatch/ui/BitmapToStitches.kt

```diff
@@ -164,3 +162,21 @@ private fun savePesAfterSelection(context: Context, result: ActivityResult, byte
     }
 }
 
+//@Preview(showBackground = true)
+//@Composable
```

Comment:
needs a cleanup

Reply:
It does. Do we need it for following commits? 

If not, I'd prefer to just have it deleted. (Otherwise I will have forgotton what I want to do with it....)